.github/PULL_REQUEST_TEMPLATE

```diff
@@ -0,0 +1,46 @@
+# Proposed change
+
+<!--- 
+Please include a summary of the change and which issue is fixed in the form of fix #XX for auto-linking.
+
+- Include relevant motivation and context
+- List any dependencies that are required for this change.
+--->
+
+## Type of change
+
+<!--- Please delete options that are not relevant for your change --->
```

Comment:
**Optional! This is how Conventional Commits handles this:**

Instead of deleting options, consider requiring that the subject of the PR and/or commit should denote the type of change, i.e. either `fix: .....` or `feat: ....` and breaking changes should either be:
- marked by adding two blank lines after the commit message then adding `BREAKING CHANGE:` and a description of what is breaking
- marked by adding a `!` after the type of change in the commit subject, i.e. `fix! blah blah` or `feat! blah blah` with a description of what is broken in the commit message.

The benefit of this is that you can scan `git log` to determine if you need to bump the patch (`fix`), minor (`feat`) or major (`breaking change` or `!`) version.